Packs/Palo_Alto_Networks_Enterprise_DLP/ReleaseNotes/1_2_0.md

```diff
@@ -0,0 +1,16 @@
+
+#### Integrations
+##### Palo Alto Networks Enterprise DLP
+
+- Added support for authenticating with Client ID and Client Secret
+- Added support for adjusting idle time between incident pulling based on value returned from DLP service
```

Comment:
Due to the change in the credentials are breaking changes 
please create the release notes with the -bc flag 
`demisto-sdk update-release-notes .... -bc`
and please add a notes about the breaking changes
for example see [BC json file](Packs/AWS-GuardDuty/ReleaseNotes/1_2_6.json)

Reply:
Updated release notes